flash_examples/custom_task.py

```diff
@@ -0,0 +1,116 @@
+from typing import Any, List, Tuple
+
+import numpy as np
+import torch
+from pytorch_lightning import seed_everything
+from sklearn import datasets
+from sklearn.model_selection import train_test_split
+from torch import nn
+
+import flash
+from flash.data.auto_dataset import AutoDataset
+from flash.data.process import Postprocess, Preprocess
+
+seed_everything(42)
+
+
+class CustomPostprocess(Postprocess):
+
+    THRESHOLD = 14.72
+
+    def predict_per_sample_transform(self, pred: Any) -> Any:
+        if pred > self.THRESHOLD:
+
+            def send_slack_message(pred):
+                print(f"This prediction: {pred} is above the threshold: {self.THRESHOLD}")
+
+            send_slack_message(pred)
+        return pred
+
+
+class LinearRegression(flash.Task):
+
+    postprocess_cls = CustomPostprocess
```

Comment:
I'd add it in the constructor to show that is not needed for any specific reason to be created inside the object or any flash magic happens here

Reply:
Yes, DataPipeline can use this attribute name to instantiate directly. Wee will work on cleaning this slightly later.